pkg/actions/addon/mocks/IAMRoleCreator.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.53.4. DO NOT EDIT.
+// Code generated by mockery v2.53.3. DO NOT EDIT.
```

Comment:
looks like you used an older version on mockery. Would you mind bumping it and rerun this code-gen once?

Reply:
Oh yeah strange. Not sure how that happened. Will update